recipes/grass/meta.yaml

```diff
@@ -0,0 +1,101 @@
+{% set version = "7.4.1" %}
+
+package:
+  name: grass
+  version: {{ version }}
+
+source:
+  url: https://grass.osgeo.org/grass74/source/grass-7.4.1.tar.gz
+  sha256: 560b8669caaafa9e8dbd4bbf2b4b4bbab7dca1cc46ee828eaf26c744fe0635fc
+  patches:
```

Comment:
Could you please put some explanation why these patches are necessary?

Reply:
:+1:

It can be really hard to debug/update this recipe later. A comment per patch would be idea.

Reply:
OK. Will do.

I believe the issue is that there are tests that are run as part of the build process. In doing this, shared libraries are tested without being installed and so because they are still in the build tree, there are path problems finding various libraries.